_includes/manuals/2.2/1.2_release_notes.md

```diff
@@ -16,14 +12,252 @@ aren't familiar with all the methods Foreman provides. Documentation is availabl
+
+### Release notes for 2.2.0
+#### API
+* Sprockets complaining about graphiql assets not being precompiled for production ([#30484](https://projects.theforeman.org/issues/30484))
+* Request to /api/statuses returns 500 ISE when a smart proxy has "logs" feature disabled ([#30205](https://projects.theforeman.org/issues/30205))
+* PowerStatus of host in GraphQL ([#30107](https://projects.theforeman.org/issues/30107))
+* Support fetching all records in API ([#29909](https://projects.theforeman.org/issues/29909))
+* graphql: add queries and mutations for settings ([#28920](https://projects.theforeman.org/issues/28920))
+
+#### Authentication
+* 404 \`AuthSourceLdap not found Please try to update your request\` when trying to update auth source ([#30319](https://projects.theforeman.org/issues/30319))
+
+#### BMC
+* Associate BMC with proxies ([#28357](https://projects.theforeman.org/issues/28357))
+
+#### Compute resources - VMware
+* API Create vmware ComputeResource fails with "Datacenter can't be blank" ([#30620](https://projects.theforeman.org/issues/30620))
+
+#### Compute resources - oVirt
+* Can't add RHEV with APIv3 through Hammer ([#30372](https://projects.theforeman.org/issues/30372))
+* Keyboard layout is not working properly  ([#30231](https://projects.theforeman.org/issues/30231))
+* ERF42-0258 [Foreman::Exception]: &lt;uuid&gt; is not valid, enter id or name ([#30195](https://projects.theforeman.org/issues/30195))
+* Problem when assigning quota in ovirt ([#30175](https://projects.theforeman.org/issues/30175))
+* fix rails 6 deprecation warnings.  ([#29716](https://projects.theforeman.org/issues/29716))
+* Creating invalid image within compute resource is possible ([#29702](https://projects.theforeman.org/issues/29702))
+* Foreman lists unrelated RHV storage domains using v4 API ([#29283](https://projects.theforeman.org/issues/29283))
+
+#### DB migrations
+* make Plugin#pending_migration check plugin migrations instead of core ones ([#29498](https://projects.theforeman.org/issues/29498))
+
+#### DHCP
+* iPXE Embedded loader does not put token to the DHCP filename option ([#27879](https://projects.theforeman.org/issues/27879))
+
+#### Development tools
+* update container builds to latest base image / ruby /nodejs ([#30499](https://projects.theforeman.org/issues/30499))
+
+#### Facts
+* Extract import_facts entry point from host class and create HostFactImporter ([#30127](https://projects.theforeman.org/issues/30127))
+* Use facet features in reported data facet ([#30126](https://projects.theforeman.org/issues/30126))
+
+#### Foreman modules
```

Comment:
I've often wondered if we should prefix this categories from subprojects with the subproject. That would be `Installer - Foreman modules` instead.

Reply:
good idea! I'll add that.